src/components/BorrowerProfile/SupporterDetails.vue

```diff
@@ -1,28 +1,54 @@
 <template>
 	<router-link
-		class="tw-flex tw-flex-col md:tw-flex-row md:tw-items-center tw-text-black"
-		:to="`${displayType === 'lenders' ? '/lender/' + this.publicId : '/team/' + this.publicId}`"
-		:v-kv-track-event="configureTracking"
+		class="
+			tw-flex tw-flex-col
+			md:tw-flex-row md:tw-items-center
+			tw-no-underline hover:tw-no-underline
+			tw-mb-1"
+		:to="linkPath"
+		v-kv-track-event="configureTracking"
 	>
-		<!-- eslint-disable-next-line max-len -->
-		<div class="tw-block md:tw-inline-block md:tw-flex-none md:tw-mr-2 tw-w-[135px] md:tw-w-[150px] lg:tw-w-[84px] xl:tw-w-[96px]">
+		<div class="
+			tw-block md:tw-inline-block
+			md:tw-flex-none
+			md:tw-mr-2
+			tw-max-w-[160px] md:tw-w-[84px] lg:tw-w-[84px] xl:tw-w-[96px]
+			tw-mb-1 md:tw-mb-0"
+			style="line-height: 0; /* global property affects images within anchors - override required */"
```

Comment:
@ryan-ludwig It would be useful to have at least the tw-leading-0 activated for cases where we need to 0 our line-height.